scenario/askhubadvisor.go

```diff
@@ -758,33 +745,74 @@ func (settings AskHubAdvisorSettings) Execute(sessionState *session.State, actio
 	if label == "" {
 		label = "hubadvisorquery"
 	}
+
+	host, err := connection.GetRestUrl()
+	if err != nil {
+		actionState.AddErrors(errors.WithStack(err))
+		return
+	}
+
+	reqHeaders := map[string]string{"x-qlik-client-capability": "static"}
+	reqOptions := session.DefaultReqOptions()
+	reqOptions.ExpectedStatusCode = []int{http.StatusOK, http.StatusCreated}
+	reqOptions.ContentType = "application/json"
+	reqOptions.FailOnError = true
+
+	var wg sync.WaitGroup
+	wg.Add(3)
```

Comment:
more clear, especially in the future when/if the amount of requests are changed if instead a wg.Add(1) is used before each async request.